index.md

```diff
@@ -1,63 +1,200 @@
 ---
-title: Flutter is a mobile app SDK for iOS and Android
+title: Welcome to Flutter
 sidebar: home_sidebar
 layout: page
 ---
 
-Flutter is a new project to help developers build high-performance, high-fidelity, mobile apps for
-iOS and Android from a single codebase.
+Flutter is a new project to help developers build high-performance, high-fidelity, 
+mobile apps for iOS and Android from a single codebase.
 
 {% include note.html content="_Flutter is an early-stage open-source project_. We hope you try Flutter out and send us [feedback](mailto:flutter-dev@googlegroups.com)." %}
 
-## New to Flutter
+## What is Flutter?
 
-See these topics to quickly start building cross-platform mobile apps with Flutter.
+Flutter provides an easy and productive way to build and deploy high-performance 
+mobile apps for Android and iOS. The goal is to deliver apps that feel natural 
+on different platforms, and we embrace differences like scrolling behavior, 
+overscroll behaviors, and more. 
```

Comment:
It seems weak to mention scrolling twice.  Maybe typography is a better second example?